src/site/markdown/server/manage-webadmin.md

```diff
@@ -294,7 +294,9 @@ Response codes:
 ### Create a user
 
 ```
-curl -XPUT http://ip:port/users/usernameToBeUsed -d '{"password":"passwordToBeUsed"}'
+curl -XPUT http://ip:port/users/usernameToBeUsed \
+  -d '{"password":"passwordToBeUsed"}' \ 
+  -H "Content-Type: application/json"
```

Comment:
don't you need also `Accept` header?

Reply:
No it's not needed, ContentType is enough